packages/devextreme-react/src/core/__tests__/component.test.tsx

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
This is to skip currently non-compilable tests for which I would prefer to spare their git history.
`// @ts-nocheck` to silence the linter, skipping in `jest.config.js` instead of `describe.skip()` to avoid compilation attempts.